util/src/main/java/org/datacommons/util/PlaceSeriesSummary.java

```diff
@@ -40,8 +46,8 @@ public void extractSeriesFromNode(McfGraph.PropertyValues node) {
     Hasher hasher = Hashing.farmHashFingerprint64().newHasher();
     hasher.putString(vres.toString(), StandardCharsets.UTF_8);
     Long hash = hasher.hash().asLong();
-    Map<Long, SeriesSummary> seriesSummaryMap =
-        svSeriesSummaryMap.getOrDefault(vres.getStatVarDcid(), new HashMap<>());
+    svSeriesSummaryMap.computeIfAbsent(vres.getStatVarDcid(), k -> new HashMap<>());
```

Comment:
Doesn't computeIfAbsent return the value? That will save another map lookup.

Reply:
done